.circleci/continue_config.yml

```diff
@@ -61,17 +61,66 @@ commands:
 
   build_test:
     description: "build and test csharp code!"
+    parameters:
+      threshold:
+        default: 50
+        type: integer
     steps:
       - checkout
+      - run:
+          name: Get report generator
+          command: dotnet tool install dotnet-reportgenerator-globaltool --version 5.1.10 --tool-path Tools/
       - run:
           name: Restore solution dependencies and tools
-          command: dotnet restore CircleCiTestFunction/CircleCiTestFunction.sln
+          command: dir "*.csproj" -Recurse | %{dotnet restore $_.FullName}
       - run:
-          name: Build solution
-          command: dotnet build CircleCiTestFunction/CircleCiTestFunction.sln
+          name: Build solution 
+          command: dir "*.csproj" -Recurse | %{dotnet build $_.FullName}
       - run:
-          name: "Running tests"
-          command: dotnet test CircleCiTestFunction/CircleCiTestFunction.sln
+          name: Run tests and build coverage report
+          command: |
+
+            # run dotnet test on all projects ending in .UnitTests.csproj
+            # There should be one [Name].UnitTests.csproj for every [Name].csproj
+            $test_projs = Get-ChildItem -Include "*.UnitTests.csproj" -Recurse
```

Comment:
The test projects will have only Tests at the end not UnitTests. Do you want me to rename them to .UnitTests.csproj?